randovania/games/fusion/logic_database/Sector 2 TRO.txt

```diff
@@ -398,9 +399,11 @@ Extra - room_id: [13, 46]
           All of the following:
               Hi-Jump
               After Boss Zazabi Defeated or Wall Jump (Beginner)
+          # Freeze Zoros
           Stand On Frozen Enemies (Beginner) and Can Freeze Enemies
```

Comment:
This needs to be removed, it is not possible anymore to freeze and stand on the zoros